llama_stack/providers/utils/inference/openai_compat.py

```diff
@@ -527,10 +527,13 @@ async def convert_message_to_openai_dict_new(
     async def _convert_message_content(
         content: InterleavedContent,
     ) -> Union[str, Iterable[OpenAIChatCompletionContentPartParam]]:
-        async def impl():
+        async def impl(content: InterleavedContent):
```

Comment:
can you rename this just so it is clear there isn't a name collision anywhere later unintentionally?